src/main/java/com/parkhomovsky/bookstore/controller/BookController.java

```diff
@@ -1,24 +1,36 @@
 package com.parkhomovsky.bookstore.controller;
 
 import com.parkhomovsky.bookstore.dto.BookDto;
+import com.parkhomovsky.bookstore.dto.BookSearchParameters;
 import com.parkhomovsky.bookstore.dto.CreateBookRequestDto;
 import com.parkhomovsky.bookstore.service.BookService;
 import java.util.List;
 import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
 @RequiredArgsConstructor
 @RestController
-@RequestMapping(value = "/api/books")
+@RequestMapping(value = "/books")
```

Comment:
```suggestion
@RequestMapping(value = "/api/books")
```

Reply:
But I have this in application.properties:
server.servlet.context-path=/api

Reply:
good, didn't notice that.